Add log lines of update history in the scripts for WAFS processing enhancement

YaliMao-NOAA opened the pull request:
The main changes are:
1) Add change logs in relevant workflow scripts for wafs processing update.
2) Drop the fix of triggering post_anl job for reliability due to concern of GFS operation.

YaliMao-NOAA committed:
1. By updating GTG tag, modify the GTG source code: when there is a high wind (>500 m/s), send out a warning WITHOUT error message
and let GTG computation continue;
2. modify the script so the WAFS failure will not affect generation of GFS

YaliMao-NOAA committed:
Merge remote-tracking branch 'upstream/release/gfs_v16' into post_wafs.v16

YaliMao-NOAA committed:
Add log lines of update history in the scripts for the fix:
modify the script so the WAFS failure will not affect generation of GFS

WenMeng-NOAA commented on scripts/exglobal_atmos_pmgr.sh:
```diff
@@ -71,7 +73,7 @@ do
   sleep 10
   icnt=$((icnt + 1))
-  if [ $icnt -ge 1080 ]
+  if [ $icnt -ge 1001 ]
```
@lgannoaa May I ask the reason of changing 1080 into 1001? I recalled the 1080 was added based on NCO GFS V16 testing. @KateFriedman-NOAA may chime in.

WenMeng-NOAA replied:
@lgannoaa Got it. Thanks for clarifying.

lgannoaa commented:
Hi Wen,

Arun authorized task is to add comments to the file.
Please only pick up the comments session from my file.

Lin

On Sat, Jun 12, 2021 at 11:51 AM WenMeng-NOAA ***@***.***>
wrote:

> *@WenMeng-NOAA* commented on this pull request.
> ------------------------------
>
> In scripts/exglobal_atmos_pmgr.sh
> <https://github.com/NOAA-EMC/EMC_post/pull/330#discussion_r650406413>:
>
> > @@ -71,7 +73,7 @@ do
>
>    sleep 10
>    icnt=$((icnt + 1))
> -  if [ $icnt -ge 1080 ]
> +  if [ $icnt -ge 1001 ]
>
> @lgannoaa <https://github.com/lgannoaa> May I ask the reason of changing
> 1080 into 1001? I recalled the 1080 was added based on NCO GFS V16 testing.
> @KateFriedman-NOAA <https://github.com/KateFriedman-NOAA> may chime in.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EMC_post/pull/330#pullrequestreview-682334779>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AI62YCLK76VIBWMVJSARRUTTSN675ANCNFSM46RKIBIQ>
> .
>

WenMeng-NOAA commented:
@YaliMao-NOAA Please commit the changes of comment part in exglobal_atmos_pmgr.sh only as:
#  Script name:         exglobal_atmos_pmgr.sh
#
#  This script monitors the progress of the gfs_post job
#  Script history log:
#  2021-05-25   Lin Gan: Ensure loganl.txt file created by atmos_analysis_calc before release atmos_post_anl job.
#

YaliMao-NOAA committed:
Insert the comments from Lin to the history log of exglobal_atmos_pmgr.sh:
    Ensure loganl.txt file created by atmos_analysis_calc before release atmos_post_anl job

YaliMao-NOAA commented:
@WenMeng-NOAA exglobal_atmos_pmgr.sh now only includes the changes of comment part.

WenMeng-NOAA reviewed (approved):
Look good to me.

WenMeng-NOAA commented:
@YaliMao-NOAA They look good to me. Thanks!

YaliMao-NOAA committed:
Drop the fix of post_anl job trigger for reliability for GFS operation concern
